frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -124,11 +134,13 @@ def process_exception(self, request, exception, spider):
     def open(self, spider):
         log.msg('Starting frontier', log.INFO)
         if not self.frontier.manager.auto_start:
-            self.frontier.start()
+            scrapy_kwargs = {'spider': spider}
+            self.frontier.start(**scrapy_kwargs)
```

Comment:
As discussed, we don't support at the moment passing custom settings using dash API. Also that change is not enough because we need to visualize job settings and dash also does not support that.

However, we discussed yesterday with @shaneaevans and he agreed to use spider arguments as now. Using this has the advantage that it is already there, so we really don't need to add another way to configure jobs. Also, that does not imply to pass the spider to the backend. I suggested you either to instantiate the frontier in the open() method of the scrapy scheduler instead in the init(), or override frontier settings in open() method, thus allowing the scheduler to pass some configuration dict in the spider. He agreed that either alternative is good. And this does not violate neither encapsulation principle (spider is not passed to the backend) nor adds extra awareness of the backend to the spider (spider is used only as a via to pass arguments, nothing else)